tests/unit/PlansTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace PagarMe\Endpoints\Test;
+
+use PagarMe\Client;
+use PagarMe\Endpoints\Plans;
+use PagarMe\Test\Endpoints\PagarMeTestCase;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\Psr7\Response;
+
+final class PlanTest extends PagarMeTestCase
+{
+    public function planProvider()
+    {
+        return [[[
+            'plan' => new MockHandler([
+                new Response(200, [], self::jsonMock('PlanMock'))
+            ]),
+            'planList' => new MockHandler([
+                new Response(200, [], self::jsonMock('PlanListMock'))
+            ])
+        ]]];
+    }
+
+    /**
+     * @dataProvider planProvider
+     */
+    public function testPlanCreate($mock)
+    {
+        $requestsContainer = [];
+        $client = self::buildClient($requestsContainer, $mock['plan']);
+
+        $response = $client->plans()->create([
+            "amount" => 15000,
```

Comment:
Double quotes are for string interpolation. If you don't need it, you can always use single quotes :wink: